src/routes/LoginWithToken.tsx

```diff
@@ -71,11 +71,11 @@ export const LoginWithToken: React.FC = () => {
               >
                 personal access tokens
               </a>{' '}
-              and create one with the{' '}
+              and create one with the three scopes{' '}
```

Comment:
Any worry that "three" could become outdated?

Reply:
Absolutely!  Saving future us heartache, thank you!

Updated to use AUTH_SCOPES array length